pandas/io/json.py

```diff
@@ -61,6 +66,22 @@ def __init__(self, obj, orient, date_format, double_precision,
         if orient is None:
             orient = self._default_orient
 
+        self.is_json_table_schema = orient == 'json_table_schema'
+        if self.is_json_table_schema:
+            self.schema = to_json_schema(obj)
```

Comment:
So I would call this ``orient='table'`` as what you are calling ``json_table_schema``.

I would also make this a super-class of ``FrameWriter`` (you would intercept this in ``to_json`` to create that class, alternaively use a ``.create`` class method on ``Writer`` to dispatch to the correct class (may be a bit cleaner).